src/FixedDataTable.js

```diff
@@ -959,7 +999,8 @@ var FixedDataTable = createReactClass({
     }
 
     var lastScrollToRow  = oldState ? oldState.scrollToRow : undefined;
-    if (props.scrollToRow != null && (props.scrollToRow !== lastScrollToRow || viewportHeight !== oldViewportHeight)) {
+    var rowHeightChanged = props.scrollToRow != null ? this._scrollHelper.getRowHeightChanged(props.scrollToRow) : 0
```

Comment:
What's the scenario where this change is needed?  I think you maybe added this to support expanding, collapsing, and expanding a row again.  For that behavior, maybe we could set scrollToRow to null when collapsing the row.  See my motivation below.

I'd prefer to keep the act of scrolling to a row to a single point in time, so that it takes effect, but is then ignored until changed.  This seems geared at resetting the scroll to the row when it's height changes, which could have adverse consequences to normal scroll behavior.  e.g. I set scroll to row 10, then scroll down manually to row 200, then I update my rowHeightGetter function, and then grid unexpectedly moves me back up to row 10.

If we can remove this like I think we can, let's also remove the getRowHeightChanged method from scrollHelper.